docs/build/guides/account-linking-with-dapper.md

```diff
@@ -0,0 +1,752 @@
+* Run Cadence Scripts from the frontend
+
+## Prerequisites
+
+### Next.js and Modern Frontend Development
+
+This tutorial uses [Next.js].  You don't need to be an expert, but it's helpful to be comfortable with development using a current React framework.  You'll be on your own to select and use a package manager, manage Node versions, and other frontend environment tasks.  If you don't have your own preference, you can just follow along with us and use [Yarn].
+
+### Flow Wallet
+
+You'll need a [Flow Wallet], but you don't need to deposit any funds.
+
+## Moments NFTs
+
+You'll need a [Dapper Wallet] containing some Moments NFTs, such as [NBA Top Shot] Moments.
+
+## Getting Started
+
+This tutorial will use a [Next.js] project as the foundation of the frontend.  Create a new project with:
+
+```zsh
+npx create-next-app@latest
+```
+
+We will be using TypeScript and the App Router, in this tutorial.
+
+Open your new project in the editor of your choice, install dependencies, and run the project.
+
+```zsh
+yarn install
+yarn run dev
+```
+
+If everything is working properly, you'll be able to navigate to `localhost:3000` and see the default [Next.js] page.
+
+## Flow Cadence Setup
+
+You'll need a few more dependencies to efficiently work with Cadence inside of your app.
+
+### Flow CLI and Types
+
+The [Flow CLI] contains a number of command-line tools for interacting with the Flow ecosystem.  If you don't already have it installed, you can add it with Brew:
+
+```zsh
+brew install flow-cli
+```
+
+Once it's installed, you'll need to initialize Flow in your Next.js project.  From the root, run:
+
+```zsh
+flow init --config-only
+```
+
+Next, you'll need to do a little bit of config work so that your project knows how to read Cadence files.  Install Raw Loader:
+
+```zsh
+yarn add raw-loader --dev
+```
+
+And the Flow Cadence Plugin:
+
+```zsh
+yarn add flow-cadence-plugin --dev
+```
+
+Finally, open `next.config.ts` and update it to use the plugin with Raw Loader:
+
+```tsx
+// next.config.ts
+import type { NextConfig } from "next";
+import FlowCadencePlugin from "flow-cadence-plugin";
+
+const nextConfig: NextConfig = {
+  webpack: (config) => {
+    config.plugins.push(new FlowCadencePlugin())
+
+    return config;
+  },
+};
+
+export default nextConfig;
+```
+
+## Frontend Setup
+
+We'll use the Flow Client Library [FCL] to manage blockchain interaction from the frontend.  It's similar to viem, ethers, or web3.js, but works with the Flow blockchain and transactions and scripts written in Cadence. 
+
+```zsh
+yarn add @onflow/fcl
+```
+
+Go ahead and install dotenv as well:
+
+```
+yarn add dotenv
+```
+
+### Provider Setup
+
+A fair amount of boilerplate code is needed to set up your provider.  We'll provide it, but since it's not the purpose of this tutorial, we'll be brief on explanations.  For more details, check out the [App Quickstart Guide].
```

Comment:
Nice

Reply:
This way they'll always get the most up-to-date version of that and we don't have to maintain it here and there!